services/plansProposalsFacade/plansProposalsFacade-portlet/src/test/java/com/ext/portlet/service/impl/XCoLabTest.java

```diff
@@ -1,12 +1,23 @@
 package com.ext.portlet.service.impl;
 
+import com.ext.portlet.NoSuchContestException;
```

Comment:
Most updates in this class are to not fail creation of objects if they were previously created.